Jenkinsfile

```diff
@@ -9,7 +9,7 @@ pipeline {
   }
 
   environment {
-     EBSCO_RESOURCE_MANAGEMENT_API_BASE_URL = "https://api.ebsco.io"
+     TEST_RMAPI_URL = "https://api.ebsco.io"
```

Comment:
agreed that we have to coordinate with John Malconian and hongwei to see if this Jenkins file param is being used in deployments